src/main/archetype/ui.frontend/src/main/webpack/components/content/download/scss/download.scss

```diff
@@ -0,0 +1,45 @@
+/* BEM Description
+BLOCK cmp-download
+    ELEMENT cmp-download__title
+    ELEMENT cmp-download__title-link
+    ELEMENT cmp-download__description
+    ELEMENT cmp-download__properties
+    ELEMENT cmp-download__property
+        MOD cmp-download__property--<property>
+    ELEMENT cmp-download__property-label
+    ELEMENT cmp-download__property-content
+    ELEMENT cmp-download__action
+    ELEMENT cmp-download__action-text
+*/
+
+.cmp-download {
+  &__title {
+    &-link {
+
+    }
+  }
+
+  &__description {
+
+  }
+
+  &__properties {
+
+  }
+
+  &__property {
```

Comment:
We could list property modifiers we provide out of the box for filename, size and format:
```css
.cmp-download__property--filename
.cmp-download__property--size
.cmp-download__property--format
```